### DISCUSSION TITLE
v2.2.1b5

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#525] [Announcements]
## What's Changed
* Hide mqtt password (fixes #516, #313) by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/524

**Full Changelog**: https://github.com/ESPresense/ESPresense/compare/v2.2.1b4...v2.2.1b5

<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense/releases/tag/v2.2.1b5'>v2.2.1b5</a>.</em>

**Comment by ankycooper** [comment]
Thanks for the quick turnaround, both mqtt username and password are masked, however the portal is still accessible without password.
Also have we considered OTA update password similar to esphome, preventing rouge flashing of devices.

**Reply by DTTerastar** [reply]
Arduino OTA is disabled by default.  Only keep it on long enough to update it.
